testing/install/CXX/CMakeLists.txt

```diff
@@ -4,12 +4,17 @@
 #------------------------------------------------------------------------------#
 
```

Comment:
In this file I added the regression test for the deprecated adios2::cxx11* targets

Reply:
That's a good idea :+1: 

Does it also make sense to add a test including one or more of the deprecated headers? Not sure, just asking :grinning: